timewatch/timewatch.py

```diff
@@ -67,17 +70,17 @@ def login(self, company, user, password):
     self.company = company
     self.user = user
     self.password = password
-    self.employeeid = int(BeautifulSoup.BeautifulSoup(r.text).find('input', id='ixemplee').get('value'))
+    self.employeeid = int(BeautifulSoup(r.text).find('input', id='ixemplee').get('value'))
 
     self.logger.info("successfully logged in as {} with id {}".format(self.user, self.employeeid))
 
     return r
 
   def time_to_tuple(self, t):
-    if isinstance(t, (str, unicode)):
+    if isinstance(t, str):
```

Comment:
I would add following code to the top:

    try:
      basestring
    except NameError:
      basestring = str

and use `basestring` instead of `str`.

That would support unicode string input in python2 (which isn't the case now but better safe than sorry IMO)